### PR TITLE
LPS-171650 Add endpoint to retrieve a user's ticket

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one comment. Otherwise looks good to go 👍

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -4130,6 +4147,50 @@ paths:
...
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Ticket"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/Ticket"
+            tags: ["Ticket"]
+    "/user-accounts/{userAccountId}/password-rest-ticket":
```
[the remark this answers is not in the document]
Comment (pei-jung) [reply modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml:4172]: Good catch, thank you!

[the remark this answers is not in the document]
Comment (drewbrokke) [reply modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml:4172]: Yup, anytime!

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:forward

**Comment by pei-jung** [comment]
@patriciajeanperez are the failures related?

**Comment by pei-jung** [comment]
ci:reevaluate:1357401_3302

**Review by drewbrokke — APPROVED** [review]

**Comment by patriciajeanperez** [comment]
@pei-jung Hi Pei! The failure is not related.

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by pei-jung** [comment]
@patriciajeanperez if there are no related failures, can we manually forward this to Brian?

**Comment by patriciajeanperez** [comment]
Pushed to BChan
https://github.com/brianchandotcom/liferay-portal/pull/128871

**Pull request closed without merging**